tests/drivers/build_all/input/app.overlay

```diff
@@ -26,6 +26,17 @@
 			};
 		};
 
+		kbd-matrix {
+			compatible = "gpio-kbd-matrix";
+			row-gpios = <&gpio0 0 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>,
+				    <&gpio0 0 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			col-gpios = <&gpio0 0 GPIO_ACTIVE_LOW>,
+				    <&gpio0 0 GPIO_ACTIVE_LOW>,
+				    <&gpio0 0 GPIO_ACTIVE_LOW>;
+			row-size = <2>;
+			col-size = <3>;
```

Comment:
couldn't these be inferred from len(row-gpios) and len(col-gpios)?

Reply:
not really, they are defined in the generic code while the gpios one are in this specific implementation

Reply:
I mean, the driver could make these optional in this particular case?

Reply:
...unless I duplicate the initializer and make a version that takes row and col as an argument, how does that sound?

Reply:
up to you, just thought these are a bit redundant in this case.

Reply:
alright how's it look now? more macros more better